src/AqBanking/Command/RenderContextFileToXMLCommand.php

```diff
@@ -17,7 +17,6 @@ public function execute(ContextFile $contextFile, bool $returnSimpleXml = false)
             $this->pathToAqBankingCLIBinary
             . ' export'
             . ' --ctxfile=' . escapeshellcmd($contextFile->getPath())
-            . ' --transactiontype=statement' // we are only interested in statements, not notedStatement and so on
```

Comment:
Can we make this optional using a `$filterTransactionType = null` parameter? where we can pass `statement` to filter?

Reply:
Ah now I get it - that's why you added "type" to transaction? Okay with me ...  We just need to bump the Major version then and add a note in the changelog